settings.gradle.kts

```diff
@@ -509,7 +509,7 @@ hideFromDependabot(":instrumentation:vibur-dbcp-11.0:javaagent")
 hideFromDependabot(":instrumentation:vibur-dbcp-11.0:library")
 hideFromDependabot(":instrumentation:vibur-dbcp-11.0:testing")
 hideFromDependabot(":instrumentation:wicket-8.0:javaagent")
-
+hideFromDependabot(":instrumentation:thrift:javaagent")
```

Comment:
By convention, we put the minimum supported library version in the module name; in case of thrift it should be `:instrumentation:thrift-0.14:javaagent`